test/test_specifications.py

```diff
@@ -43,101 +53,86 @@ def _working_directory_fix():
         os.chdir(current_working_directory / "..")
 
 
-def test_run_will_be_reduced_when_all_rules_are_ment(specification: InstrumentSpecification, job_request) -> None:
-    """
-    Test that a job_request.will_reduce remains set to true when its relevant specification is met
-    :param specification: specification fixture
-    :param job_request: JobRequest fixture
-    :return: None
-    """
-    mock_rule = Mock()
-    specification._rules = [mock_rule, mock_rule, mock_rule]
-    specification.verify(job_request)
-    assert job_request.will_reduce
+@mock.patch("rundetection.specifications.InstrumentSpecification._load_rules_from_api")
+def test_instrument_specification_tries_to_loads_api_from_db(load_rules_from_api):
```

Comment:
Should be spelt "load" to make sense grammatically.

Also should these tests be missing docstrings?